lfc-scheduler/Makefile

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARCHS = amd64 arm64
+COMMONENVVAR=GOOS=$(shell uname -s | tr A-Z a-z)
+BUILDENVVAR=CGO_ENABLED=0
+
+LOCAL_REGISTRY=localhost:5000/scheduler-plugins
+LOCAL_IMAGE=kube-scheduler:latest
+LOCAL_CONTROLLER_IMAGE=controller:latest
+
+# RELEASE_REGISTRY is the container registry to push
+# into. The default is to push to the staging
+# registry, not production(k8s.gcr.io).
+RELEASE_REGISTRY?=docker.io/annadreal
```

Comment:
Shouldn't this be keptnsandbox?

Reply:
do we have one or should we make it? 🤔

Reply:
I commented it out for now, we will make one soon

Reply:
I think keptnsandbox already exists ...